test/integration/patients/worklist/schedule.js

```diff
@@ -137,14 +126,26 @@ context('schedule page', function() {
             attributes: _.extend(_.sample(this.fxPatients), {
               first_name: 'LongTest',
               last_name: 'PatientName',
-              id: 1,
+              id: '1',
             }),
           },
         );
 
         return fx;
       })
-      .routeAction()
+      .routeAction(fx => {
```

Comment:
I think a bulk of the flakiness was caused by one of the actions in the list being overwritten by this request. If the due date in the new data was before the current day's date, it would break the action order checks